polaris/ocean/tasks/isomip_plus/topo/remap.py

```diff
@@ -0,0 +1,140 @@
+    MPAS mesh
+
+    """
+    def __init__(self, component, name, subdir, config, topo_map, experiment):
+        """
+        Create the step
+
+        Parameters
+        ----------
+        component : polaris.Component
+            The component the step belongs to
+
+        name : str
+            The name of the step
+
+        subdir : str
+            The subdirectory in the test case's work directory for the step
+
+        config : polaris.config.PolarisConfigParser
+            A shared config parser
+
+        topo_map : polaris.ocean.tasks.isomip_plus.topo_map.TopoMap
+            The step for creating a mapping files, also used to remap data
+            from the MPAS mesh to a lon-lat grid
+
+        experiment : {'ocean1', 'ocean2', 'ocean3', 'ocean4'}
+            The ISOMIP+ experiment
+        """
+        super().__init__(component=component, name=name, subdir=subdir)
+
+        self.set_shared_config(config, link='isomip_plus.cfg')
+
+        geom_filenames = dict(
+            ocean1='Ocean1_input_geom_v1.01.nc',
+            ocean2='Ocean2_input_geom_v1.01.nc',
+            ocean3='Ocean3_input_geom_v1.01.nc',
+            ocean4='Ocean4_input_geom_v1.01.nc')
+
+        if experiment not in geom_filenames:
+            raise ValueError(f'No input geometry for experiment {experiment}')
+
+        self.add_input_file(filename='topography.nc',
+                            target=geom_filenames[experiment],
+                            database='isomip_plus')
+        self.add_dependency(topo_map, name='topo_map')
+        self.add_output_file('topography_remapped.nc')
+
+    def run(self):
+        """
+        Run this step of the test case
+        """
+        self._preprocess()
+        self._remap()
+        self._rename()
+        self._renormalize()
+
+    def _preprocess(self):
+        ice_density = self.config.getfloat('isomip_plus', 'ice_density')
+
+        with (xr.open_dataset('topography.nc') as ds_in):
+            if 't' in ds_in.dims:
+                ds_in = ds_in.chunk({'t': 1})
+            ds_in['iceThickness'] = ds_in.upperSurface - ds_in.lowerSurface
+            ds_in.iceThickness.attrs['description'] = 'ice thickness'
+            ds_in.iceThickness.attrs['units'] = 'm'
+
+            gravity = constants['SHR_CONST_G']
+            ds_in['landIcePressure'] = (ice_density * gravity *
+                                        ds_in.iceThickness)
+            ds_in.iceThickness.attrs['description'] = \
+                'pressure at the ice base'
+            ds_in.iceThickness.attrs['units'] = 'Pa'
+
+            ds_in = ds_in.rename({'floatingMask': 'landIceFloatingFraction',
+                                  'groundedMask': 'landIceGroundedFraction',
+                                  'openOceanMask': 'openOceanFraction',
+                                  'lowerSurface': 'landIceDraft'})
+            ds_in['oceanFraction'] = \
+                (ds_in.bedrockTopography < 0.).astype(float)
```

Comment:
I remember that this is what we have used for the thin film cases. Can you point out where the grounded land ice part of the domain gets culled out when there is not a thin film? Or does that not happen?

Reply:
It will get handled in the initial condition by setting `maxLevelCell` to 0, so proper culling doesn't happen.

Reply:
That sounds good to me. Thanks for clarifying!

Reply:
I don't believe there is a to-do item here so I'm going to resolve.